routes/raker.router.js

```diff
@@ -0,0 +1,194 @@
+const express = require('express');
+const {db} = require('../server/db')
```

Comment:
🚫 **[eslint]** <[import/newline-after-import](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/newline-after-import.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected 1 empty line after require statement not followed by another require.
```suggestion
const {db} = require('../server/db')

```